### ISSUE TITLE
Make cursor optionally bigger

### ISSUE THREAD

**Issue by n0toose** [xournalpp/xournalpp#4722]
**Is your feature request related to a problem? Please describe.**
- I can't draw on a large display using a medium-sized pen, as I cannot find/keep missing the cursor and have to draw on the page to find it again.
- Even on a laptop screen, I can barely find the cursor and find myself doing this every now and then.

**Describe the solution you'd like**
Make the of the cursor that shows when drawing 50% or 100% bigger, even if that doesn't correspond to the actual size of the cursor. This may not be optimal for drawing (where a lot of precision is required), but it would be absolutely awesome for note-taking (where some precision is required, especially with a small tablet and multiple windows open).

---

P.S. Your work has saved my life, thank you for everything that you do.

**Comment by Artim96** [comment]
Yes, right this. It does help changing the cursor from a tiny spot to the pen symbol, but even with the colored circle around it, it's really difficult to see where the cursor is exactly. This also makes it more difficult to precisely draw so the new drawing seamlessly connects to an older one.

For making the cursor easier to find, I'd suggest something like showing some visual queue when hitting a specific key, like you can set on Windows with the Ctrl key. But in general a cursor that's generally better visible while still being able to see where exactly you'll be writing/drawing would be great.

**Comment by PeterRugg** [comment]
Yes! This has been an issue for me as well, in an otherwise perfect tool for marking work!

**Comment by Dacastror** [comment]
I also had a problem with this. For my needs it was enough to change the settings in preferences (menu view/preferences) in the view cursor section:

<img src="https://raw.githubusercontent.com/Dacastror/images/main/preferences_xournal.png" width=550px% title="Xournal++ on GNU/Linux"/>

There I selected "big pen icon" or sometimes I use "system cursor"

**Comment by Dacastror** [comment]
I also had a problem with this. For my needs it was enough to change the settings in preferences (menu edit/preferences) in the view cursor section:

<img src="https://raw.githubusercontent.com/Dacastror/images/main/preferences_xournal.png" width=550px% title="preferences"/>

There I selected "big pen icon" or sometimes I use "system cursor"

Or in the case of wanting to use the "small dot icon" and make it more visible, it worked well for me to check the "Highlight cursor position" option:

<img src="https://raw.githubusercontent.com/Dacastror/images/main/xournal2.png" width=550px% title="preferences"/>

Anyway, I agree with the request to be able to increase the cursor size beyond the actual stroke size
